scripts/select_appointments.sql

```diff
@@ -1 +1,20 @@
 -- retrieve appointment information
+SELECT p.given_name, p.surname, a.appt_time
+FROM patients p
+INNER JOIN doctors d
+  ON p.patient_id = d.id
+INNER JOIN appointments a
+  ON d.doctor_id = a.id
+WHERE doctor_id = 6
+;
```

Comment:
Not quite. `appointments` is the join between patients and doctors. And while the order of the `INNER JOIN`s doesn't matter, you do need to use the right columns from the right tables.